Cargo.toml

```diff
@@ -297,9 +298,20 @@ repository = "https://github.com/aptos-labs/aptos-core"
 rust-version = "1.85"
 
 [workspace.dependencies]
+
+# External crate dependencies.
+# Please do not add any test features here: they should be declared by the individual crate.
```

Comment:
Why did we move these (and only these) external deps to the top?

Reply:
must be from my deps sort. Lemme revert and preserve the existing order